ImplementationGuide/markdown/Diagnose/Diagnose_Interaktionen.md

```diff
@@ -12,7 +12,7 @@ Folgende Suchparameter sind für das Bestätigungsverfahren relevant, auch in Ko
 
     Anwendungshinweise: Weitere Informationen zur Suche nach "_id" finden sich in der [FHIR-Basisspezifikation - Abschnitt "Parameters for all resources"](https://hl7.org/fhir/R4/search.html#all).
 
-1. Der Suchparameter "clinical-status" MUSS unterstützt werden:
+1. Der Suchparameter "clinical-status" MUSS, sofern das Element unterstützt wird (siehe in der Profilstruktur Festlegung zu 'Conditional Must Support'), unterstützt werden:
```

Comment:
typo: 2x "unterstützt wird/werden"

Reply:
danke @alexey-tschudnowsky, ist korrigiert.
Suggestion: da dies nur informative Änderung ohne Bezug zum Packaging, würde ich das direkt mergen wollen in die Stufe 2